workspaces/lluis/Tag.svelte

```diff
@@ -0,0 +1,37 @@
+<script lang="typescript">
+  export let isContainer = false
+  export let color = null
+  export let theme = null
+  export let size = null
+  export let isDelete = false
+  export let isRounded = false
+  export let style = ""
+</script>
+
+{#if isContainer}
```

Comment:
I wouldn't go with isContainer.

It seems like the container version has basically noting in common with the tag version.

So I'd rather have separate `<Tag />` and `<Tags />` components, which would simplify both the code and the usage

Reply:
This is now implemented in the new commit